flathunter/abstract_crawler.py

```diff
@@ -193,6 +196,71 @@ def resolve_geetest(self, driver):
             driver.refresh()
             raise
 
+    @backoff.on_exception(wait_gen=backoff.constant,
+                        exception=CaptchaUnsolvableError,
+                        max_tries=3)
+    def resolve_awsawf(self, driver):
+        """Resolve AWS WAF Captcha"""
+
+        # Intercept background network traffic via log sniffing
+        sleep(2)
+        logs_raw = driver.get_log("performance")
```

Comment:
This is some deep magic right here. Well done for finding you're way around this.